calibration2024/calibration2024.tex

```diff
@@ -0,0 +1,149 @@
+% Copyright (C) 2024 - Michaël Baudin
```

Comment:
you can move the folder into /userday2024

Reply:
You are right: fixed.